amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/raml/Raml08DocumentParser.scala

```diff
@@ -105,6 +115,7 @@ case class Raml08DocumentParser(root: Root)(implicit override val ctx: RamlWebAp
 
   private def parseSchemaDeclarations(map: YMap, parent: String): Unit = {
     map.key("schemas").foreach { e =>
+      ctx.addDeclarationKey(DeclarationKey(AnyShapeModel, e))
```

Comment:
why not ShapeModel instead of AnyShapeModel?

Reply:
Based on the fact that the Raml08TypeParser returns an Option[AnyShape]. Do you think ShapeModel would be more appropriate?